de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/AuthTestResource.java

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.server.resources;
+
+import javax.annotation.security.DenyAll;
+import javax.annotation.security.PermitAll;
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+import de.dlr.sc.virsat.server.auth.ServerRoles;
+
+@Path("/auth")
+public class AuthTestResource {

```

Comment:
You are right the intention behind this resource was just to have a resource to test with. I don't think we need an extra resource for this in production, because failed authentication returns the distinct "forbidden" http status. 

I am not sure if we should include this resource via DI, because we would then also have to adjust the dynamic repository binding in some way to also register that test resource. I actually wanted to delete this resource once we have the actual resources and test authentication there. But I am not sure, so do you still think DI is the right thing to do here?

Reply:
This resource will be removed once we have the actual resources where we test authentication.